sdk/core/azure-core/inc/azure/core/internal/strings.hpp

```diff
@@ -11,21 +11,31 @@
 #include <algorithm>
 #include <string>
 
-namespace Azure { namespace Core { namespace _internal { namespace Strings {
+namespace Azure { namespace Core { namespace _internal {
 
-  bool LocaleInvariantCaseInsensitiveEqual(const std::string& lhs, const std::string& rhs) noexcept;
-  std::string const ToLower(const std::string& src) noexcept;
-  unsigned char ToLower(const unsigned char src) noexcept;
-
-  struct CaseInsensitiveComparator
+  /**
+   * @brief Extend the functionality of std::string by offering static methods for string
+   * operations.
+   *
+   */
+  struct StringExtensions
```

Comment:
nit: I'd make this a class, because you rather think of it as a <static> class. Structs are more about data members.

Reply:
doesn't need any private member, it is meant to be use just a bag of functions :)